sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -1304,6 +1305,28 @@ def test_enet_sample_weight_consistency(
     assert_allclose(reg1.coef_, reg2.coef_, rtol=1e-6)
 
 
+@pytest.mark.parametrize("fit_intercept", [True, False])
+@pytest.mark.parametrize("sparse_container", [None] + CSC_CONTAINERS)
+def test_enet_cv_sample_weight(fit_intercept, sparse_container, global_random_seed):
+    rng = np.random.RandomState(global_random_seed)
+    rng = np.random.RandomState(0)
```

Comment:
Why reseed with 0? We want to this test to be seed insensitive and pass for all admissible `global_ransom_seed` values (between 0 and 99).

```suggestion
```

To test locally you can use:

```
$ SKLEARN_TESTS_GLOBAL_RANDOM_SEED="all" pytest -k test_enet_cv_sample_weight -vl
```

Once they all pass locally, check that they also all pass on the CI by pushing an empty commit with a coded commit message as documented in this pinned issue:

https://github.com/scikit-learn/scikit-learn/issues/28959

Sometimes there are platform specific behavior and we want to detect, investigate and fix any such failures prior to merging the PR.

Reply:
Hmmm O.K. I tested and it passes locally let me try launch a coded commit